### PR TITLE
Adds a round end animation

### PR DESCRIPTION
**Pull request by Cyberboss**
https://youtu.be/UzDmiL-Ue-4

Video slightly out of date, animation time changed from 2 to 3 seconds, HUD also fades out.

:cl: Cyberboss
imageadd: New round end animation. Inspired by @Iamgoofball 
/:cl:

### CODE REVIEW

**Comment by Qbopper** [comment]
neat

**Comment by PKPenguin321** [comment]
cooool

**Comment by PKPenguin321** [comment]
wait if you were to var edit the title screen icon before this happened would the edited image come up

**Comment by Jordie0608** [comment]
Yes

**Comment by Cyberboss** [comment]
I'm gonna go against that idea. The point of this PR is to make the server reboot transition less jarring, and you can't change the title screen for the next round while the current one is going on.

**Comment by optimumtact** [comment]
At round end does this wait until the server is actually restarting or does it fire as soon as the shuttle docks?

**Comment by chanoc1** [comment]
what does this add waahh stop changing my legacy hard cut to menu

**Comment by Cyberboss** [comment]
@optimumtact It runs on world/Reboot

**Comment by RandomMarine** [comment]
**/TG/STATION13**

**DIRECTED BY**
**M. SPACE SHYLAMAJ**

**Comment by Cyberboss** [comment]
Play of the game when?

**Comment by Niknakflak** [comment]
its ok I guess

**Comment by PKPenguin321** [comment]
Only thing that is kind of sketch is sometimes I will use that last brief moment to examine something or take a screenshot before the round finishes resetting but this is cool so whatever

**Comment by Incoming5643** [comment]
a fade in is cool for now but down the line I want the full assortment of screen transitions.

Friggen STAR WIPES

File: code/world.dm [hunk 1]
```diff
@@ -269,6 +269,17 @@ var/last_irc_status = 0
...
+	
+	var/titlescreen = TITLESCREEN
+	if(!titlescreen)
+		titlescreen = "title"
+
+	for(var/thing in clients)
+		if(thing)
+			var/client/C = thing
+			C.screen.Cut()
+			C.screen += new /obj/screen/splash
```
Comment (Cyberboss) [inline code/world.dm]: Can someone tell me if there's a way to put this on top without deleting the hud?

Comment (RemieRichards) [inline code/world.dm]: stupidly high plane value.

**Comment by ChangelingRain** [comment]
If we ever get more than one titlescreen, or titlescreens based on map, this is going to be *incredibly* jarring when swapping to a new map.

**Comment by RemieRichards** [comment]
@ChangelingRain they are per map! it's just nobody uses it...

**Comment by Cyberboss** [comment]
I'm touching map loading right now, so I'll fix that there

**Comment by Cyberboss** [comment]
I'm using this in that PR, so when this is merged, please don't squash/rebase.

**Pull request merged**